webui/react/src/components/ColumnPickerMenu.tsx

```diff
@@ -63,6 +66,12 @@ interface ColumnTabProps {
   onHeatmapSelectionRemove?: (id: string) => void;
 }
 
+export const formatColumnKey = (col: ProjectColumn): string => {
```

Comment:
i think we wanna store all columns with types based on the discussion 2 weeks ago.

Reply:
just confirming -- as the only affected columns are run metadata columns, we should be okay to limit the type encoding to run metadata columns.

Reply:
As far as i know, hyperparameter would have the same names with different types too. i think its safer to encode all columns for the future and consistency.